Azure/communication-ui-library-ios
# [Feature] added leave call confirmation drawer list

## Purpose
Added leave call confirmation drawer/list into app. (iPhone and iPad, light and dark model)
![iPhone leave call (light)](https://user-images.githubusercontent.com/90668345/162853453-a3364e6f-09f9-43a7-8045-01d2c87e1c31.png)
![iPad leave call (dark)](https://user-images.githubusercontent.com/90668345/162853466-ab67cebd-8929-4132-9b65-b3e47d0cc0a5.png)

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
* Launch the app, join either group or Teams meeting
* tap leave call button
* follow **What to Check** section below

## What to Check
Verify that the following are valid
* check if the leave call drawer appears correctly in iPhone (as drawer/list) and in iPad (as pop over)
* check if the Leave button and the Cancel button behave correctly
* check if the leave call drawer appears correctly in both orientation and both theme modes (light and dark)

## Other Information
